packages/compass-crud/src/stores/crud-store.js

```diff
@@ -1252,8 +1230,29 @@ const configureStore = (options = {}) => {
   if (options.globalAppRegistry) {
     const globalAppRegistry = options.globalAppRegistry;
 
-    globalAppRegistry.on('instance-created', ({ instance }) => {
```

Comment:
This event would never fire because this is a scoped plugin and the instance already exists by the time this gets executed.